### ISSUE TITLE
Wrong price on games with the same name

### ISSUE THREAD

**Issue by JoiNNN** [jshackles/Enhanced_Steam#1399]
There are these 2 games
http://store.steampowered.com/app/252850/Streamline/
http://store.steampowered.com/app/393660/Streamline/
when requesting the price history from ITAD it returns price data for both, but ES picks the 1st, causing a price mismatch.

To fix this we can either pass the issue to ITAD, or simply do an appid/bundleid check when a Steam link is returned.

**Comment by JoiNNN** [comment]
@jshackles any advice on this? I think it would better if fixed on ITAD's side.

**Comment by jshackles** [comment]
Yes I will definitely pass this (and any future games like this we find) to ITAD.  The network tab shows that it's submitting the correct SubID for each game, but the data for app 252850 is returning incorrect data from their API.

Strangely, I'm not able to reproduce an instance where their API is returning both.

**Comment by Skifoid** [comment]
Another games with same names and wrong price history:
http://store.steampowered.com/app/716330/VOID/
http://store.steampowered.com/app/503710/Void/

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
